server.js

```diff
@@ -100,9 +100,23 @@ module.exports = function(opts, callback) {
     });
   }
 
-  app.listen(process.env.PORT || opts.port, process.env.HOST || opts.bind, function() {
-    console.log("Listening at http://%s:%d/", this.address().address, this.address().port);
+  var handler = process.env.SOCKET || opts.socket || process.env.PORT || opts.port;
+  var server = app.listen(handler, process.env.HOST || opts.bind, function() {
+    var endpoint;
+    if (opts.socket) {
```

Comment:
I think it would be logical to indicate this in `process.env` too, but this is optional.